test/integration/targets/wait_for/files/latin1.txt

```diff
@@ -0,0 +1,4 @@
+premi�re is first
```

Comment:
Since this file is not UTF-8 encoded it should be generated when the test runs. Perhaps using a Python script with escaped sequences, or base 64 encoded.

Reply:
I can do that, it was in a script in expect, and I just wrote it out statically.